File: C++/059. Vogais2/Vogais2.cpp
#include <iostream>
int main()
{
    char A;
    std::cin >> A;
    if (A == 'a' || A == 'e' || A == 'i' || A == 'o' || A == 'u' ||
        A == 'A' || A == 'E' || A == 'I' || A == 'O' || A == 'U') 
    {
        std::cout << "SIM" << std::endl;
        return 0;
    }
    else 
    {
        return 0;
    }
    return 0;
}

File: Java/059. Vogais2/Vogais2.java
import java.util.Scanner;
public class Vogais2
{
    public static void main(String[] args) 
    {
        Scanner scan = new Scanner(System.in);
        char X = scan.next().charAt(0);
        if (X == 'a' || X == 'e' || X == 'i' || X == 'o' || X == 'u' || 
            X == 'A' || X == 'E' || X == 'I' || X == 'O' || X == 'U')
        {
            System.out.println("SIM");
        }
        else
        {
            scan.close();
            return;
        }
        scan.close();   
    }
}